sdk/schemaregistry/schema-registry/src/schemaRegistryClient.ts

```diff
@@ -40,9 +46,25 @@ export class SchemaRegistryClient implements SchemaRegistry {
     credential: TokenCredential,
     options: SchemaRegistryClientOptions = {}
   ) {
-    const pipeline = createPipeline(options, credential);
     this.endpoint = endpoint;
-    this.client = new GeneratedSchemaRegistryClient(endpoint, { ...pipeline, endpoint });
+
+    const internalPipelineOptions: InternalPipelineOptions = {
+      ...options,
+      ...{
+        loggingOptions: {
+          logger: logger.info,
+          additionalAllowedHeaderNames: ["x-ms-correlation-request-id", "x-ms-request-id"]
```

Comment:
these weren't specified in the previous pipeline. Do we want them?

Reply:
Apparently not, I mindlessly copied it from TA. Removed it.